### ISSUE TITLE
[Bug] Alarm doesn't loop alarm sound

### ISSUE THREAD

**Issue by aaronhsmine** [PC-Logix/OpenSecurity#147]
**Describe the bug**
The alarm block only plays the alarm sound once instead of looping it. After playing the sound once, it needs to be deactivated before it can be reactivated to play the sound once more. Tested with default and custom alarm sounds.

**In which environment did the Bug appear?**
Singleplayer

**To Reproduce**
1. Place down alarm block
2. Set up some kind of OpenComputers Computer
3. Connect alarm block to Computer
4. Set alarm sound and activate alarm using the Computer
5. Alarm block plays sound once, then stops

**Expected behavior**
Alarm should loop the alarm sound indefinetly until deactivated.

**Screenshots / Code Snippet**
Code used for activating the alarm (executed both from a file and from the OpenOS lua shell):
```lua
alarm = component.os_alarm
alarm.setAlarm("alarmsound")
alarm.activate()
```

**Minecraft:**
 - Mod Version: OpenSecurity-1.12.2-1.0-39
 - Minecraft Version: Minecraft 1.12
 - Forge Version: Forge 14.23.5.2847

**Comment by aaronhsmine** [comment]
Just a quick question, this issue was received, right?

**Comment by ben-mkiv** [comment]
yes, but im not able to reproduce it reliable. Most of the time it loops fine for me, so im not quite sure what is causing the issue.

**Comment by aaronhsmine** [comment]
I experimented a bit more and it also works fine for me on a new world. I also couldn't find anything which might be related to this in the log or debug file while playing on the world where it didn't work. It might have something to do with the other mods i use, but I'm closing this issue for now.

**Comment by ben-mkiv** [comment]
Well there was some issue with the Alarm for sure, as it didn't play for me when reloading a world where it played before.

Let me leave this bug opened as reminder to myself to look at it at some point... :)

**Comment by CaitlynMainer** [comment]
Fixed in f7f68d9fb21d7844a6116a979608933b3a0d8e0f
